src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,289 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private static final char ADDITION = '+';
+    private static final char SUBTRACTION = '-';
+    private static final char MULTIPLICATION = '*';
+    private static final char DIVISION = '/';
+    private static final char POWER = '^';
+    private static double DELTA = 0.0001;
+    private static Calculator calculator;
+    private static double actual;
+    private static double expected;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    void calculate_additionPositivePositive_Ok() {
```

Comment:
better rename tests, for example:
```suggestion
    void calculate_additionTwoPositiveOperands_Ok() {
```